packages/atomic/src/components/commerce/search-box-suggestions/atomic-commerce-search-box-instant-products/atomic-commerce-search-box-instant-products.tsx

```diff
@@ -89,7 +89,7 @@ export class AtomicCommerceSearchBoxInstantProducts
    */
   @Prop() public ariaLabelGenerator?: AriaLabelGenerator;
 
-  public componentWillLoad() {
+  public componentDidLoad() {
```

Comment:
Aux4: lifecyle/racing issue around dispatchSearchBoxSuggestionsEvent.
Test without this line.